libkbfs/keybase_daemon_rpc.go

```diff
@@ -267,6 +267,18 @@ func (*KeybaseDaemonRPC) HandlerName() string {
 	return "KeybaseDaemonRPC"
 }
 
+func (k *KeybaseDaemonRPC) registerProtocolLocked(p rpc.Protocol) error {
```

Comment:
This should accept a server too.

Reply:
Yup! I was thinking the same thing. The latest commit does exactly that.